.circleci/config.yml

```diff
@@ -38,14 +38,6 @@ workflows:
       # for PRs
       - yarn/update-cache:
           <<: *only_development
-      - yarn/run:
```

Comment:
cc @zephraph removing this for now. SSH'ing into a Circle CI box reveals the 'scope' this is getting run in has changed and doesn't seem to be able to pick up diffs (from the PR to master).

Reply:
Sounds good